features/addition.feature

```diff
@@ -4,17 +4,18 @@ Feature: Addition
   So I can know their sum
 
   Scenario: Adding numbers to get a sum
-    When I add two numbers
-    Then I get the sum
+    When I add two numbers together
+    Then I get an integer
+    Then I get the correct sum
```

Comment:
Seems like an unnecessary test. It doesn't hurt anything, but I don't know if you gain anything from it. Correctness will be tested by later steps. Just my 2 cents.